src/gantry/screens.py

```
@@ -666,6 +675,55 @@ def action_focus_search(self) -> None:
+    def _apply_active_filter(self) -> None:
+        """Filter _raw_resources by _active_filter and repopulate the table."""
+        filtered = k8s.filter_resources(self._raw_resources, self._active_filter)
+        # Keep _resource_data in sync with what the table shows.
+        self._resource_data = filtered
+        try:
+            table: ResourceTable = self.query_one("#resource-table", ResourceTable)
+            columns, keys = self._get_columns_and_keys()
+            if columns:
+                table.populate_resources(filtered, columns, keys)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)